### PR TITLE
Feature: Frontend Next JS Boilerplate

### PR DESCRIPTION
**Pull request by ckvignesh**
Hi @evedes,

I've created the frontend folder that contains the cleaned boilerplate.

### CODE REVIEW

File: frontend/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
```
Comment (evedes) [inline frontend/pages/index.js:2]: Let's remove the Home.module to have a clean first boilerplate?

File: frontend/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
```
Comment (evedes) [inline frontend/pages/index.js:6]: Remove all the styles for now. We'll add tailwind later.

File: frontend/styles/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.container {
```
Comment (evedes) [inline frontend/styles/Home.module.css:1]: Let's remove this file for now? ❤️

**Review by evedes — CHANGES_REQUESTED** [review]
Hey, good job here! Just left some minor comments ❤️

File: frontend/pages/index.js (hunk 1 above)
Comment (ckvignesh) [reply frontend/pages/index.js:2]: Done.

File: frontend/pages/index.js (hunk 2 above)
Comment (ckvignesh) [reply frontend/pages/index.js:6]: Done.

File: frontend/styles/Home.module.css (hunk 3 above)
Comment (ckvignesh) [reply frontend/styles/Home.module.css:1]: Done

**Comment by ckvignesh** [comment]
Hi @evedes, I've worked on the changes you mentioned. Can you please review them?

**Review by evedes — APPROVED** [review]
Great job Vi, @ckvignesh ! 🔥

**Comment by ckvignesh** [comment]
> Great job Vi, @ckvignesh ! fire

Thank you @evedes !!

**Pull request merged**